app/services/sign_in/well_known.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module SignIn
+  module WellKnown
+    class Generator
```

Comment:
If you look at the other services in this folder you can see that the class names are in the format `{Thing}{Action}`.
So your class name should be `WellKnownGenerator`. 

Additionally, your file structure always needs to match `/module(s)/class`